test/unit/utils.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { expect } from 'chai';
+import { parseUri, getUri, UriParts } from '../../lib/utils/utils';
```

Comment:
maybe it's worth extracting these uri util functions to a `uriUtils` file, and have this test file name accordingly.

Reply:
That would be fine with me, I'll make the change.